drivers/usb/device/usb_dc_stm32.c

```diff
@@ -235,7 +235,7 @@ static int usb_dc_stm32_clock_enable(void)
 	 * that instead.  Example reference manual RM0360 for
 	 * STM32F030x4/x6/x8/xC and STM32F070x6/xB.
 	 */
-#if defined(RCC_HSI48_SUPPORT)
+#if defined(RCC_HSI48_SUPPORT) || defined(CONFIG_SOC_SERIES_STM32WBX)
```

Comment:
STM32WBX does not have `RCC_HSI48_SUPPORT` defined because all the forms of it include the HSI48 support, AFAICT. I changed this `if` block to account for this so the clock got configured in software properly for this platform.

Better way to do this?

Reply:
This is fine. We might hit issues if some WB variants show up w/o HSI48, but with current WB Cube package, I don't see a better way.